.ci/scripts/apm-server-update-beats.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -euox pipefail
+
+source ./_beats/dev-tools/common.bash
+
+jenkins_setup
+
+cleanup() {
+  rm -rf $TEMP_PYTHON_ENV
+  make stop-environment fix-permissions
+}
+trap cleanup EXIT
+
+go get github.com/elastic/apm-server/vendor/github.com/kardianos/govendor
+RACE_DETECTOR=1 make update-beats
+
+eval "$(gvm $(cat ./_beats/.go-version))"
+
+RACE_DETECTOR=1 make clean check testsuite apm-server
```

Comment:
I am wondering why this file needs to be copied here, as it is part of the APM Server repository that gets checked out anyways in this test script.

Reply:
you are right it is not needed, the script is running in the apm-server context so it does not need to copy it.